src/ring_theory/tensor_product.lean

```diff
@@ -706,6 +706,18 @@ rfl
 @[simp] lemma map_comp_include_right (f : A →ₐ[R] B) (g : C →ₐ[R] D) :
   (map f g).comp include_right = include_right.comp g := alg_hom.ext $ by simp
 
+lemma map_range (f : A →ₐ[R] B) (g : C →ₐ[R] D) :
+  (map f g).range = (include_left.comp f).range ⊔ (include_right.comp g).range :=
+begin
```

Comment:
Is there a `linear_map.range` version of this statement too? Can the alg_hom version be proved easily from that one?

Reply:
The issue is that `include_left` and `include_right` are defined in terms of tensoring with `1`, which isn't possible in the linear case.

Reply:
I think the closest equivalent in the linear case would be a lemma that we already have https://github.com/leanprover-community/mathlib/blob/7ca68f8e03953c4b365b2bb1f32d5df8a80def2d/src/linear_algebra/tensor_product.lean#L607